tools/go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/stacklok/minder/tools
 
-go 1.21
-
-toolchain go1.21.3
+go 1.21.5
```

Comment:
Do we need the patch number here? thought we could make due with just major and minor

Reply:
I ran `go mod tidy` and it did this.

Reply:
Oh!  This is pulling the go version from `./go.mod` because of the `replace` directive...